lib/v26/OC.d.ts

```diff
@@ -1,11 +1,96 @@
 declare namespace Nextcloud.v26 {
 
+    interface FilesNavigationColumn {
```

Comment:
Is it for 26 and 28 ? Even if it's supported, might be some corner cases we didn't see, especially since they're on old `@nc/files`

I would keep it quiet and only go for 28+ :see_no_evil:

Reply:
Was added there (checked the source code^^)

Reply:
yes, but the specs changed
The View from @nc/files beta8 is different than the one from, say, beta 10 and same for final.
I would not document versions older than 28

Reply:
Well it is public API (OCP) on 26 and 27. But I think not that useful? Changed the PR to only provide for 28.